unaliased-setup.py

```diff
@@ -52,14 +52,16 @@
     "nextcord",
     "nextcord.types",
     "nextcord.ui",
+    "nextcord.ui.select",
     "nextcord.webhook",
+    "nextcord.ext.application_checks",
     "nextcord.ext.commands",
```

Comment:
If we are adding new features to the discord alias, also add `slash_utils`
```suggestion
    "nextcord.ext.commands",
    "nextcord.ext.slash_utils",
```